src/util/noteValue.ts

```diff
@@ -1,16 +1,18 @@
 import State from '../@types/State'
 import ThoughtId from '../@types/ThoughtId'
-import findDescendant from '../selectors/findDescendant'
 import { firstVisibleChild } from '../selectors/getChildren'
-import getThoughtById from '../selectors/getThoughtById'
+import resolveNotePath from '../selectors/resolveNotePath'
+import thoughtToPath from '../selectors/thoughtToPath'
+import head from './head'
 
 /** Gets the value of a thought's note. Returns null if the thought does not have a note. */
 const noteValue = (state: State, id: ThoughtId) => {
-  const noteId = findDescendant(state, id, '=note')
-  if (!noteId) return null
-  const noteThought = getThoughtById(state, noteId)
-  if (noteThought?.pending) return null
-  return firstVisibleChild(state, noteId!)?.value ?? null
+  // Try to resolve path (checks =children first, then regular =note/=path)
+  const targetPath = resolveNotePath(state, thoughtToPath(state, id))
```

Comment:
`thoughtToPath` doesn't work with nested context views. You'll have to modify this function to take a `Path` instead of a `ThoughtId`.